src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/mapper/interventions/BreachMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.probation.courtcaseservice.restclient.communityapi.mapper.interventions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Component;
+import uk.gov.justice.probation.courtcaseservice.service.model.Breach;
+import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiNsi;
+import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiNsiResponse;
+
+@Component
+public class BreachMapper {
+    public Breach breachFrom(CommunityApiNsi nsi) {
```

Comment:
Would expect a unit test class direct on this. It's probably effectively tested by integration tests higher up additionally but please add